src/CreateNewPage/index.js

```diff
@@ -41,6 +54,9 @@ class CreateItemPage extends Component {
     return !this.itemId.value;
   };
 
+  handleTextChange = e => {
+
```

Comment:
this is required when the Component is controlled, we keep it emtpy for now depending for what is required next :)